nixos/modules/services/home-automation/home-assistant.nix

```diff
@@ -157,7 +157,8 @@ in {
       default = [];
       example = literalExpression ''
         with pkgs.home-assistant-custom-components; [
-          prometheus-sensor
+          prometheus_sensor
```

Comment:
Let me know if I got this wrong / mis-understood the new format @mweinelt - this is just the docs / sample.

Reply:
Understood - will do!